__tests__/intl/index.spec.js

```diff
@@ -1071,5 +1072,15 @@ describe('intl duck', () => {
       await expect(getLocalePack('xx-Xxxx-XX')).resolves.toBe('xx');
       await expect(getLocalePack('yy-Yyyy-YY')).resolves.toBe('yy-Yyyy');
     });
+    it('should only load each langpack once', async () => {
```

Comment:
Is it possible to mock `localePack()` so that you can verify it's not called again, which I think is the real intent of this code?

Reply:
That is what I wanted to do initially, but that would require a restructure of these tests, which I wasn't keen on doing for this small change 😄

Reply:
Fair enough!